projects/1.20/assets/cybernetic-system/cybernetic_system/lang/zh_cn.json

```
@@ -0,0 +1,899 @@
+  "gui.cybernetic_system.gui_diolog_dna.label_receive": "[接受]",
+  "gui.cybernetic_system.gui_diolog_golden_apple.label_an_apple_an_apple_now_bring": "苹果！苹果！现在...给我带DNA过来！",
+  "gui.cybernetic_system.gui_diolog_golden_apple.label_continue": "[继续]",
+  "gui.cybernetic_system.gui_diolog_nano_machin_yes.label_continue": "[继续]",
+  "gui.cybernetic_system.gui_diolog_nano_machin_yes.label_now_now_bring_me_an_apple": "现在...现在...给我带个苹果！",
+  "gui.cybernetic_system.gui_diolog_nano_machin_yes.label_the_golden_apple": "一个金苹果！",
+  "gui.cybernetic_system.gui_diolog_nano_machin_yes.label_you_brought_it_you_brought_it": "你拿来了！你拿来了！",
+  "gui.cybernetic_system.gui_diolog_start_quest.label_bring_it_bring_me_a_nanonano_m": "带过来！带给我一个纳米——纳米机器！",
+  "gui.cybernetic_system.gui_diolog_start_quest.label_okey": "好吧...",
+  "gui.cybernetic_system.gui_diolog_start_quest.label_wonderful": "棒  极  了！",
+  "gui.cybernetic_system.gui_diolog_start_quest.label_yeah_yeah_what_do_you_need": "哼啊，你需要什么？",
+  "gui.cybernetic_system.gui_diolog.label_how_to_become_a_cyborg": "怎么成为赛博人？",
+  "gui.cybernetic_system.gui_diolog.label_who_are_you": "你是谁？",
+  "gui.cybernetic_system.gui_diolog.label_yeah_yeah_what_do_you_need": "哼啊，你需要什么？",
+  "gui.cybernetic_system.gui_furniture_manu.tooltip_blue_energy_glass": "蓝色充能玻璃",
+  "gui.cybernetic_system.gui_furniture_manu.tooltip_computer": "计算机",
+  "gui.cybernetic_system.gui_furniture_manu.tooltip_green_energy_glass": "绿色充能玻璃",
+  "gui.cybernetic_system.gui_furniture_manu.tooltip_medium_container": "中型保险箱",
+  "gui.cybernetic_system.gui_furniture_manu.tooltip_red_energy_glass": "红色充能玻璃",
+  "gui.cybernetic_system.gui_furniture_manu.tooltip_sensor_door_blue": "传感门 (蓝色)",
+  "gui.cybernetic_system.gui_furniture_manu.tooltip_sensor_door_green": "传感门 (绿色)",
+  "gui.cybernetic_system.gui_furniture_manu.tooltip_sensor_door_red": "传感门 (红色)",
+  "gui.cybernetic_system.gui_furniture_manu.tooltip_small_container": "小型保险箱",
+  "gui.cybernetic_system.gui_furniture_manu.tooltip_soda_machine": "苏打汽水机",
+  "gui.cybernetic_system.gui_genetics_terminal.tooltip_combat": "战斗",
+  "gui.cybernetic_system.gui_genetics_terminal.tooltip_live": "生存",
+  "gui.cybernetic_system.gui_genetics_terminal.tooltip_mining": "挖掘",
+  "gui.cybernetic_system.gui_guide.tooltip_access_level": "访问等级",
+  "gui.cybernetic_system.gui_haracteristics_inde.label_no": "否",
+  "gui.cybernetic_system.gui_haracteristics_inde.label_yes": "是",
+  "gui.cybernetic_system.gui_haracteristics_inde.tooltip_access_level": "访问等级",
+  "gui.cybernetic_system.gui_haracteristics_inde.tooltip_charge": "能量",
+  "gui.cybernetic_system.gui_haracteristics_inde.tooltip_cybernation": "赛博化",
+  "gui.cybernetic_system.gui_haracteristics_inde.tooltip_dna": "DNA",
+  "gui.cybernetic_system.gui_haracteristics_inde.tooltip_maximum_health": "最大生命值",
+  "gui.cybernetic_system.gui_identification_tablet.label_10_dna": "10 DNA",
+  "gui.cybernetic_system.gui_identification_tablet.label_30_level": "30 Level",
```

```suggestion
  "gui.cybernetic_system.gui_identification_tablet.label_30_level": "30级",
```
这里的level指的是经验等级吗，如果是需翻译